remote-config/src/main/kotlin/com/rakuten/tech/mobile/remoteconfig/api/PublicKeyFetcher.kt

```diff
@@ -0,0 +1,34 @@
+package com.rakuten.tech.mobile.remoteconfig.api
+
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.json.Json
+import java.io.IOException
+
+internal class PublicKeyFetcher(private val client: ConfigApiClient) {
+
+    fun fetch(keyId: String): String {
+        val response = client.fetchPath("keys/$keyId")
+
+        if (!response.isSuccessful) {
+            throw IOException("Unexpected response when fetching public key: $response")
+        }
+
+        val body = response.body()!!.string()
```

Comment:
Is this force-unwrapping safe here?

Reply:
Yeah, if `response.isSuccessful()` is true, then the body should never be null. I should put a comment here about that.